python/cudf_polars/cudf_polars/dsl/ir.py

```diff
@@ -325,18 +330,22 @@ def __init__(
             raise NotImplementedError(
                 "Read from cloud storage"
             )  # pragma: no cover; no test yet
-        if any(
-            str(p).startswith("https:/" if POLARS_VERSION_LT_131 else "https://")
-            for p in self.paths
-        ):
+        if (
+            any(str(p).startswith("https:/") for p in self.paths)
+            and POLARS_VERSION_LT_131
+        ):  # pragma: no cover; no test yet
```

Comment:
Is this just untested because it's https (not just plain http)?

Reply:
Polars passed the wrong URI before version 1.31. To avoid having to support the malformed URI in pylibcudf/polars, I'm just raising unconditionally. I think this is fine because we're eventually going to do a lower bound up of polars in cudf anyway.